### DISCUSSION TITLE
How to add an inverter

### DISCUSSION THREAD

**Discussion by gd35** [tbnobody/OpenDTU#118] [Q&A] [answered]
Should I enter the inverter id or should it be discovered automatically ?

**Comment by tbnobody** [comment, marked as answer]
Go to Settings --> Inverter Settings and add your inverter. Auto discovery is not possible

**Reply by gd35** [reply]
OK I will have thus to remove the inverter from its position to get access to its number. Thanks for your prompt answer.
